### PR TITLE
Fix: Delete old amplitude_benefits keys

### LINKED ISSUE #3133: Amplitude: Delete old custom Airflow DAG operator code/tables
## User story / feature request

Delete old Amplitude Airflow/view files:
- ingest_amplitude_raw_dev.JSONL

### Acceptance Criteria

There should be fewer tables in GCP. These should not be there:
- cal-itp-data-infra-staging.amplitude.benefits_events
- cal-itp-data-infra-staging.views.amplitude_benefits_events
- cal-itp-data-infra.views.amplitude_benefits_events

### Notes

@angela-tran Anything else that can be deleted?

### PR DESCRIPTION
**Pull request by machikoyasuda**
# Description

Resolves #3133

_Describe your changes and why you're making them. Please include the context, motivation, and relevant dependencies._

- Delete `CALITP_AMPLITUDE_BENEFITS_API_KEY`, `CALITP_AMPLITUDE_BENEFITS_SECRET_KEY` keys, as they are no longer used. Remove them from `.env.sample` file.
- Deletes `airflow/.env.sample` file entirely.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## How has this been tested?

_Include commands/logs/screenshots as relevant._

## Post-merge follow-ups

_Document any actions that must be taken post-merge to deploy or otherwise implement the changes in this PR (for example, running a full refresh of some incremental model in dbt). If these actions will take more than a few hours after the merge or if they will be completed by someone other than the PR author, please create a dedicated follow-up issue and link it here to track resolution._

- [ ] No action required
- [x] Actions required (specified below):

Remove these now un-used tables from Google Cloud Storage:
- `ingest_amplitude_raw_dev.JSONL` 
- `cal-itp-data-infra-staging.views.amplitude_benefits_events`
- `cal-itp-data-infra.views.amplitude_benefits_events` 

Remove these keys from GCS Secret Manager:
image

### CODE REVIEW

**Review by angela-tran — APPROVED** [review]
👍

**Comment by angela-tran** [comment]
@lauriemerrell Do we need to remove these keys from Secrets Manager as well? I'm just going off what's written  in https://github.com/cal-itp/data-infra/pull/1741. I don't actually know if they're even still in there or if Secrets Manager is still a thing you use 😐

**Comment by angela-tran** [comment]
Ok actually I don't have permission to Destroy the secrets

File: airflow/.env.sample [hunk 1]
```diff
@@ -1,2 +1 @@
-CALITP_AMPLITUDE_BENEFITS_API_KEY=api-key
-CALITP_AMPLITUDE_BENEFITS_SECRET_KEY=secret-key
+
```
Comment (machikoyasuda) [inline airflow/.env.sample:1]: @evansiroky @lauriemerrell Pre-commit does not like that this file is now blank. Should I add a comment in here so it's not blank? Or is it okay to delete this file entirely?

**Comment by machikoyasuda** [comment]
@evansiroky @lauriemerrell We have 2 post-merge actions we need to take. We don't have permissions to destroy the secret and I'm not sure if we have permissions to remove these now un-used tables. Are these something you have permission to do?

## Remove these now un-used tables from Google Cloud Storage:

    ingest_amplitude_raw_dev.JSONL
    cal-itp-data-infra-staging.views.amplitude_benefits_events
    cal-itp-data-infra.views.amplitude_benefits_events

## Remove these keys from GCS Secret Manager:
image


cc @angela-tran

File: airflow/.env.sample (hunk 1 above)
Comment (lauriemerrell) [reply airflow/.env.sample:1]: I think it's safe to delete this file entirely!

**Comment by lauriemerrell** [comment]
Thanks @machikoyasuda and @angela-tran!

I wrote #3188 for our actions here and am submitting it for prioritization in a future sprint -- let us know if these should be tackled urgently.

One clarification (feel free to update the ticket text in #3188 directly as well): I am not sure I understand `ingest_amplitude_raw_dev.JSONL` -- is this a specific file?

**Comment by angela-tran** [comment]
@lauriemerrell That is referring to Google Cloud Storage buckets we used to use but no longer do. This is from when we used to use it: https://github.com/cal-itp/data-infra/pull/1279/files

I only see an `ingest_amplitude_raw_prod` bucket, so I'm not sure what happened to the dev one.

**Comment by lauriemerrell** [comment]
Ah ok thanks @angela-tran ! And just to double check, do you want to delete that prod bucket and its contents, or leave it? We often leave deprecated data sitting in GCS just in case it's ever somehow needed (also since it's also not very discoverable/accessible there so there's low risk of it being stumbled upon), but may be able to delete it if you need.

File: airflow/.env.sample (hunk 1 above)
Comment (machikoyasuda) [reply airflow/.env.sample]: Thanks @lauriemerrell - Resolved here: https://github.com/cal-itp/data-infra/pull/3185/commits/46a4757fa159bfebef5cb4cbffe6e4fc992e0045

**Review by lauriemerrell — APPROVED** [review]

**Comment by angela-tran** [comment]
> Ah ok thanks @angela-tran ! And just to double check, do you want to delete that prod bucket and its contents, or leave it? We often leave deprecated data sitting in GCS just in case it's ever somehow needed (also since it's also not very discoverable/accessible there so there's low risk of it being stumbled upon), but may be able to delete it if you need.

All the data in it exists in a different form now in a BigQuery table, so I think we are good to delete it. Mainly we are wanting to clean up these old sets of data to reduce confusion when debugging an issue with the current code. It's definitely not urgent to remove it, so whatever prioritization makes sense for y'all is fine. Thanks!

**Review by angela-tran — APPROVED** [review]

**Pull request merged**